### ISSUE TITLE
Plugin is reporting a false positive error for string concatenation

### ISSUE THREAD

**Issue by tomas-sedlak-s1** [linux-china/jetbrains-just-plugin#33]
For the following code the plugin is incorrectly reporting an error (example taken from the [just docs](https://github.com/casey/just?tab=readme-ov-file#variables-and-substitution)):
```
tmpdir  := `mktemp -d`
version := "0.2.7"
tardir  := tmpdir / "awesomesauce-" + version
tarball := tardir + ".tar.gz"
```

This is how it looks like in the IDE:
![Screenshot 2024-10-02 at 10 16 44](https://github.com/user-attachments/assets/b19ba999-3c9e-4d65-accb-3e76ceacd26b)

Plugin version I'm using: 0.5.3

**Comment by seguri** [comment]
Same here. I think it appeared after upgrading to 0.5.3. I have:

```
python_dir := "venv/" + if os_family() == "windows" { "Scripts" } else { "bin" }
python := python_dir + if os_family() == "windows" { "/python.exe" } else { "/python3" }
```

And it no longer works:

```
JustToken.COMMENT, JustToken.KEYWORD_ALIAS, JustToken.KEYWORD_EXPORT, JustToken.KEYWORD_IMPORT, JustToken.KEYWORD_MOD, JustToken.KEYWORD_SET, JustToken.NEW_LINE, JustToken.OPEN_BRACKET, JustToken.RECIPE_NAME, JustToken.SHEBANG or JustToken.VARIABLE expected, got '+'
```

**Comment by linux-china** [comment]
Fixed with version 0.6.0. Please try it later.

![image](https://github.com/user-attachments/assets/1547c806-8861-4356-bf17-e5117b2fdaed)
